governance/polls/Budget Ratification Poll for Modify TechOps Core Unit DAI Budget (MIP40c3-SP86) - November 14, 2022.md

```diff
@@ -0,0 +1,72 @@
+  result_display: instant-runoff-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Approve reduced budget
+   2: Reject budget
+start_date: 2022-11-14T16:00:00
+end_date: 2022-11-28T16:00:00
+---
+# Budget Ratification Poll for Modify TechOps Core Unit DAI Budget (MIP40c3-SP86) - November 14, 2022
+
+The Governance Facilitators have placed a budget ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, November 14 at 16:00 UTC.
+
+**This is an instant-runoff vote.** 
+- **You may rank any number of options that you would support in an executive vote.** 
+- **Unranked options signal that you would not support the option(s) in an executive vote.**
+- **If you have no preference to any of the listed options, you should vote 'Abstain' as your first and only choice.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP40c3-SP86 being accepted and implemented in the Maker Protocol by ranking their support of the following options:
+* **Option 1:** Approve reduced budget: 2,398,988.
+* **Option 2:** Reject budget.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*The TechOps Core Unit handles system administration and technical support needs of Maker Protocol and its Core Units. TECH-001 strives to improve communication and collaboration between the developers, end users, and other MakerDAO stakeholders by applying DevOps principles to software delivery and first-class technical support. For the past year TECH-001 has been delivering infrastructure and support in an automated, safe, fast and repeatable way with the help from some of the modern Infrastructure Administration tools, while continuously monitoring and improving our process throughout.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/247f11f556ad5b2be78525c7fa6c9966da76ecec/MIP40/MIP40c3-Subproposals/MIP40c3-SP86.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP40c3SP86)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip40c3-sp86-modify-techops-core-unit-dai-budget/18290)
+## Outcomes
+
+This poll implements a **Minimum Positive Participation** value. The Minimum Positive Participation is currently set to **10,000 MKR**.
+
+**If the votes for Option 1 or Option 2 achieve a majority AND the votes for the winning option exceed 10,000 MKR, then the following actions will be taken:**
+* The MIP Editors will mark the proposal **Accepted** and the Governance Facilitators will confirm its passage on the Governance and Risk call on Wednesday, November 24.
```

Comment:
```suggestion
* The MIP Editors will mark the proposal **Accepted** and the Governance Facilitators will confirm its passage on the Governance and Risk call on Thursday, December 1.
```

Reply:
I know this is silly, but I've always had problems with Wednesday-Thursday... Something like the pull-push thing...